index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html lang=fr>
+    <head>
+        <meta charset="utf-8">
+        <title>dataviz 33</title>
+        
+        <meta name="description" content="....">
+
+        <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
+        
+        <script defer src="client/js/browserify-bundle.js"></script>
+
+        <link rel="stylesheet" href="https://rawgit.com/DavidBruant/better-defaults/master/defaults.css">
```

Comment:
Better to use than [normalise.css](https://www.npmjs.com/package/normalize.css)?

https://github.com/stackcss/sheetify can help bundle CSS in JS and extract them out at bundling time.